internal/provider/resource_network_test.go

```diff
@@ -287,6 +287,33 @@ func TestAccNetwork_importByName(t *testing.T) {
 	})
 }
 
+func TestAccNetwork_dhcpRelay(t *testing.T) {
+	name := acctest.RandomWithPrefix("tfacc")
+	vlanID := getTestVLAN(t)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { preCheck(t) },
+		ProviderFactories: providerFactories,
+		// TODO: CheckDestroy: ,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkConfigDHCPRelay(name, vlanID, true),
+				Check:  resource.ComposeTestCheckFunc(
+				// TODO:
```

Comment:
Incomplete?

Reply:
Yeah not quite done, was just lazy about testing so I pushed it up to get it to run 😄